packages/cozy-harvest-lib/src/components/KonnectorConfiguration/ConfigurationTab/Contracts.jsx

```diff
@@ -77,22 +85,32 @@ const customHeaderPerDoctype = {
 }
 
 const DumbContracts = ({ contracts, doctype }) => {
+  contracts = contracts.data ? contracts.data : contracts
```

Comment:
We don't have an eslint rule telling us that we should not modify the params (even after a destructuring assignment)?

Reply:
We are not modifying the params here, we are modifying the reference.

Reply:
And for the eslint rule, I searched and found : https://eslint.org/docs/rules/no-param-reassign , it's interesting I did not know that it would change the arguments. After a destructuring, I think it's safe since it boils down to assigning a new variable. I can be more explicit and assign a new variable contractData though.

Reply:
I do not think the rule is activated per default

Reply:
Done in v